plugins/woocommerce-blocks/assets/js/base/components/quantity-selector/index.tsx

```diff
@@ -50,6 +50,10 @@ export interface QuantitySelectorProps {
 	 * Whether the component should be interactable or not
 	 */
 	disabled: boolean;
+	/**
+	 * Whether the component should be editable or not
```

Comment:
Super nitpicky, but "or not" should be dropped here. I know it's used elsewhere in these docblocks but that's also wrong (no need to update them in your PR)

```suggestion
	 * Whether the component should be editable.
```